package.json

```diff
@@ -48,6 +50,7 @@
     "babel-loader": "^8.1.0",
     "babel-plugin-emotion": "^10.0.29",
     "babel-plugin-transform-es2015-modules-commonjs": "^6.26.2",
+    "caniuse-lite": "^1.0.30001154",
```

Comment:
What's with these version numbers? Looks like they are already on `1.0.30001161`

Reply:
haha right? We were getting console warnings and the suggested fix was adding this dep. This was the latest when I added it, but I can upgrade again if you want

Reply:
No, just saw it and was wondering which prompted me to look it up. They have about 400 versions in the `1.0.3x` range